src/components/ChatMessage.js

```diff
@@ -34,14 +34,41 @@ const Message = styled.div`
   max-width: 350px;
 `;
 
-export default function ChatMessage({ user, message, align }) {
-  return (
-    <StyledChatMessage className={ align }>
-      <ProfilePicture userId={user.id}/>
-      <MessageContainer>
-        <UserName>{ user.name }</UserName>
-        <Message>{ message }</Message>
-      </MessageContainer>
-    </StyledChatMessage>
-  )
+const SystemMessage = styled.div`
+  width: 100%;
+  margin: 0 0 20px 0;
+  padding: 5px 25px 5px 25px;
+  background: rgba(0, 0, 0, 0.4);
+  color: white;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+`;
+
+export default function ChatMessage({ chat, currentUser }) {
+  if (chat.type === 'chat') {
+    return (
+      <StyledChatMessage
+        className={chat.user.userId === currentUser.userId ? 'fromMe' : ''}
+      >
+        <ProfilePicture
+          userId={chat.user.userId}
+          variant={chat.user.userId === currentUser.userId ? 'right' : ''}
+        />
```

Comment:
```suggestion
  if (chat.type === 'chat') {
    const belongsToCurrentUser = chat.user.userId === currentUser.userId;
    return (
      <StyledChatMessage
        className={belongsToCurrentUser ? 'fromMe' : ''}
      >
        <ProfilePicture
          userId={chat.user.userId}
          variant={belongsToCurrentUser ? 'right' : ''}
        />
```

반복되는 부분 줄여보면 좋을 것 같아요.